### ISSUE TITLE
user guide

### ISSUE THREAD

**Issue by MatthewSteen** [open223/docs.open223.info#1]
Add user guide from https://bas-im.emcs.cornell.edu/223/223standard/-/wikis/User-Guide.

**Comment by lazlop** [comment]
@MatthewSteen Our idea for a user guide might be closer to a reference guide than a how-to guide. I'm going to try and fit things into the diataxis framework but I may need your help.

**Comment by MatthewSteen** [comment]
@lazlop sure, feel free to `git mv` the `guides/user_guide.md` if you feel it fits better with Diataxis. Should probably consider renaming to `user_reference.md` too in that case. 

Do you think a User Guide/Reference might be too long and could be more easily followed by breaking up into smaller parts?

**Comment by lazlop** [comment]
Yes I'll try breaking it up. Can you add me to the repo, or should I take out a fork?

**Comment by MatthewSteen** [comment]
Added you with write permission, let me know if you need more.

**Comment by lazlop** [comment]
I've been trying to add mermaid parsing, so that we can easily create graph diagrams inline. I just noticed that there's a difference in syntax between gh-pages and github, that should be reconciled in the future. 

This is out for decision but seems to have been reviewed https://github.com/jupyter/enhancement-proposals/pull/101

I may create another issue reminding me to update documentation when that change is made.

**Comment by MatthewSteen** [comment]
Here are the ones that were listed in the intro.md (landing/root page), which I'll remove from there.

- Guides
    - How to Model
        - Connectivity
            - Mediums 
        - Containment
        - FunctionBlocks and Controllers
        - Sensors and Actuators
        - Properties and Aspects
    - How to Access Telemtry Data
    - BACnet as example
    - How to Extend the Standard
        - Describing G36
